java/client/src/main/java/glide/api/models/commands/SetOptions.java

```diff
@@ -168,4 +169,31 @@ public String[] toArgs() {
 
         return optionArgs.toArray(new String[0]);
     }
+
+    /**
+     * Converts SetOptions into a GlideString[] to add to a {@link Command} arguments.
+     *
+     * @return GlideString[]
+     */
+    public GlideString[] toGlideStringArgs() {
+        List<GlideString> optionArgs = new ArrayList<>();
```

Comment:
Maybe it is better to call `toArgs()` and then convert strings.
But if you are planning to remote `toArgs()` method, it is OK to have code duplicated.